config/internal/apiserver/default/deployment.yaml.tmpl

```diff
@@ -49,10 +49,6 @@ spec:
               value: "{{.APIServer.AutoUpdatePipelineDefaultVersion}}"
             - name: DBCONFIG_CONMAXLIFETIMESEC
               value: "{{.APIServer.DBConfigConMaxLifetimeSec}}"
-            - name: ML_PIPELINE_VISUALIZATIONSERVER_SERVICE_HOST
-              value: "ds-pipeline-visualizationserver"
-            - name: ML_PIPELINE_VISUALIZATIONSERVER_SERVICE_PORT
-              value: "8888"
```

Comment:
when deploying a v1 dspa via testing instructions i get the following error which leads to a panic:

```
I1213 20:44:12.464136       7 main.go:89] Starting RPC server
F1213 20:44:12.464468       7 config.go:66] Please specify flag ML_PIPELINE_VISUALIZATIONSERVER_SERVICE_HOST

goroutine 104 [running]:
github.com/golang/glog.Fatalf(...)
	/go/pkg/mod/github.com/golang/glog@v1.1.0/glog.go:585
github.com/kubeflow/pipelines/backend/src/apiserver/common.GetStringConfig({0x1c35af6, 0x2c})
	/go/src/github.com/kubeflow/pipelines/backend/src/apiserver/common/config.go:66 +0xa7
main.startRpcServer(0xc00048c140)
	/go/src/github.com/kubeflow/pipelines/backend/src/apiserver/main.go:105 +0x538
created by main.main
	/go/src/github.com/kubeflow/pipelines/backend/src/apiserver/main.go:73 +0x285

SIGABRT: abort
PC=0x40a10e m=0 sigcode=0

goroutine 111 [running]:
runtime/internal/syscall.Syscall6()
	/usr/local/go/src/runtime/internal/syscall/asm_linux_amd64.s:36 +0xe fp=0xc000408f68 sp=0xc000408f60 pc=0x40a10e
```

so perhaps this is required even if the component doesn't exist?

Reply:
Fixed.